rotkehlchen/chain/evm/accounting/interfaces.py

```diff
@@ -36,3 +46,68 @@ def event_settings(self, pot: 'AccountingPot') -> dict[str, 'TxEventSettings']:
     def reset(self) -> None:  # pylint: disable=no-self-use
         """Subclasses may implement this to reset state between accounting runs"""
         return None
+
+
+class DepositableAccountantInterface(ModuleAccountantInterface):
+    """
+    Interface for protocols that allow to deposit multiple tokens in exchange for a token
+    representing the position in the pool. Examples are:
+    - Curve
```

Comment:
Shouldn't this be used in a lot more places? Isn't this kind of related to https://github.com/rotki/rotki/issues/5709?